### ISSUE TITLE
Generated Numbers not working, need to change first digit

### ISSUE THREAD

**Issue by Curtiss02** [omkarkaul/free-pizza#2]
It seems like Pizza Hut may have updated their codes to require a different starting digit, possibly based upon their store/location. 
Following on from this comment on the cheapies site where one user changed their number to a 5:

![image](https://user-images.githubusercontent.com/8885794/179157341-40ad9a32-c325-40d5-9bf5-3c215129a19a.png)

I tried with my local store (Birkenhead) and was able to get the code to work when I swapped the first digit to an 8. I tried this with multiple codes to my sucess.

I guess we would need for data to confirm whether this is the case but may be possible to create a list which maps store -> digit and allow selection on the site.

**Comment by Corruptsector** [comment]
Changing first digit to 8 worked for Upper Hutt store as well. Thanks for the Tip

**Comment by omkarkaul** [comment]
Thanks for that guys. Yeah I just tried it last night and found the Lynn Mall store starts with an 8 as well. 

I've pushed up a quick PR to solve this for now (here #3), just shows multiple possible codes each starting with 2, 5, 8. Can look into a more sophisticated solution at a later date. May look to integrate with New Relic and add a `Did this work?` button so we can run some queries and facet by store to see which ones work and which ones don't.
